assets/scss/base/_main.scss

```diff
@@ -0,0 +1,11 @@
+body{
+ color:$black;   
+ background:$white; 
+}
+.container{
+ cursor: default;
```

Comment:
Ta klasa trochę nie ma sensu.
W klasie globalnej ustawiasz kursor na domyślny, który sam z siebie jest już domyślny.
Jeśli to ma jakieś zastosowanie, to znaczy że gdzieś coś pogmatwałeś w kodzie.

Reply:
Zrobione